src/modules/categories/components/Categories.js

```diff
@@ -41,13 +41,12 @@ class Categories extends React.Component<PropsType> {
 
   getTileModels (categories: Array<CategoryModel>): Array<TileModel> {
     return categories.map(category => {
-      let cachedThumbnail = this.props.resourceCache[category.thumbnail]
+      let cachedThumbnail = this.getLocalResourceCache(category)[category.thumbnail].path
       if (cachedThumbnail) {
         cachedThumbnail = URL_PREFIX + cachedThumbnail
       }
```

Comment:
replace by: 
```js
const cachedThumbnail = this.getLocalResourceCache(category)[category.thumbnail].path
const thumbnailUrl = cachedThumbnail ? URL_PREFIX + cachedThumbnail : category.thumbnail
```

and below: `new TileModel({ [...], thumbnail: thumbnailUrl })`